rasa/nlu/featurizers/dense_featurizer/mitie_featurizer.py

```diff
@@ -17,41 +20,94 @@
 )
 from rasa.shared.nlu.constants import FEATURE_TYPE_SENTENCE, FEATURE_TYPE_SEQUENCE
 from rasa.utils.tensorflow.constants import MEAN_POOLING, POOLING
+from rasa.nlu.featurizers.dense_featurizer._mitie_featurizer import MitieFeaturizer
 
 if typing.TYPE_CHECKING:
     import mitie
 
+logger = logging.getLogger(__name__)
 
-class MitieFeaturizer(DenseFeaturizer):
-    @classmethod
-    def required_components(cls) -> List[Type[Component]]:
-        return [MitieNLP, Tokenizer]
+# TODO: This is a workaround around until we have all components migrated to
+# `GraphComponent`.
+MitieFeaturizer = MitieFeaturizer
 
-    defaults = {
-        # Specify what pooling operation should be used to calculate the vector of
-        # the complete utterance. Available options: 'mean' and 'max'
-        POOLING: MEAN_POOLING
-    }
 
-    def __init__(self, component_config: Optional[Dict[Text, Any]] = None) -> None:
-        super().__init__(component_config)
+class MitieFeaturizerGraphComponent(DenseFeaturizer2, GraphComponent):
+    @staticmethod
+    def get_default_config() -> Dict[Text, Any]:
+        return {
+            **DenseFeaturizer2.get_default_config(),
+            # Specify what pooling operation should be used to calculate the vector of
+            # the complete utterance. Available options: 'mean' and 'max'
+            POOLING: MEAN_POOLING,
+        }
+
+    def __init__(
+        self,
+        config: Optional[Dict[Text, Any]],
+        model_storage: ModelStorage,
+        resource: Resource,
+        execution_context: ExecutionContext,
+    ) -> None:
+        super().__init__(config)
+        self.config = config
 
-        self.pooling_operation = self.component_config["pooling"]
+        # graph component
+        self._model_storage = model_storage
+        self._resource = resource
+        self._execution_context = execution_context
```

Comment:
Because the component doesn't persist, I think these 3 attributes might not be required 🤔